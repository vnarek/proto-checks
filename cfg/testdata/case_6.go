package main

func main() {
	x = nil
	x := nil
	var x *int = nil
	var x *int
	x = (((nil)))
}
