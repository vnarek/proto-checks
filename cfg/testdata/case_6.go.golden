[START]
  [x = null]
    [x = null]
      [x = null]
        [x = null]
          [x = null]
